src/test/java/org/jabref/logic/importer/fetcher/ISIDOREFetcherTest.java

```diff
@@ -0,0 +1,129 @@
+                .withField(StandardField.DOI, "10.1016/j.tbs.2019.12.003")
+                .withField(StandardField.URL, "https://isidore.science/document/10670/1.hrzlqd");
+
+        Optional<BibEntry> optionalBibEntry = fetcher.performSearchById("https://isidore.science/document/10670/1.hrzlqd");
+
+        BibEntry actual = optionalBibEntry.get();
+
+        assertEquals(expected.getField(StandardField.TYPE), actual.getField(StandardField.TYPE));
+        assertEquals(expected.getField(StandardField.TITLE), actual.getField(StandardField.TITLE));
+        assertEquals(expected.getField(StandardField.AUTHOR), actual.getField(StandardField.AUTHOR));
+        assertEquals(expected.getField(StandardField.YEAR), actual.getField(StandardField.YEAR));
+        assertEquals(expected.getField(StandardField.JOURNAL), actual.getField(StandardField.JOURNAL));
+        assertEquals(expected.getField(StandardField.PUBLISHER), actual.getField(StandardField.PUBLISHER));
+        assertEquals(expected.getField(StandardField.ABSTRACT), actual.getField(StandardField.ABSTRACT));
+        assertEquals(expected.getField(StandardField.DOI), actual.getField(StandardField.DOI));
+        assertEquals(expected.getField(StandardField.URL), actual.getField(StandardField.URL));
```

Comment:
Why not

    assertEquals(Optional.of(expected), actual)

What is the difference between the entries?